src/planner/Planner.cpp

```diff
@@ -18,6 +19,17 @@ statespace::ConstStateSpacePtr Planner::getStateSpace() const
   return mStateSpace;
 }
 
+//==============================================================================
+std::unique_ptr<common::RNG> Planner::getRng() const
+{
+  if (!mRng)
+    return nullptr;
+
+  // NOTE: We usually use unique pointers to pass RNGs, so cloning is needed
+  // here.
+  return std::move(cloneRNGFrom(*mRng)[0]);
```

Comment:
This function is somewhat misleading because it returns a clone instead of returning the original RNG in this planner. I think we should either (1) return a raw pointer or (2) change the function name to something like `cloneRng()`. I'm inclined to (1) because the user could clone from a raw pointer of RNG if necessary.

If we change so, this function might need to be non-const. Otherwise, for the sake of const-correctness, it should return `const common::RNG*`, but it would prevent us from generating random values since `RNG::operator()` is not a const function.

Nit: For the future reference, `move` in `return std::move(...)` is superfluous (please see this [post](https://stackoverflow.com/a/14856553/3122234))